### PR TITLE
Creating filer plugin

### PR DESCRIPTION
**Pull request by FinalAngel**
Fixes issue #23 

Please check the issue for integrated options

**HTML Output**

image

**Admin View**

image

### CODE REVIEW

File: aldryn_bootstrap3/cms_plugins.py [hunk 1]
```diff
@@ -268,9 +273,40 @@ def render(self, context, instance, placeholder):
 plugin_pool.register_plugin(Bootstrap3SpacerCMSPlugin)
 
 
-# ########
+class Bootstrap3FileCMSPlugin(CMSPluginBase):
+    model = models.Bootstrap3FilePlugin
+    name = _("File")
```
Comment (intellectronica) [inline aldryn_bootstrap3/cms_plugins.py]: Is it OK to use the same name as the plugin you inherit from?

Comment (FinalAngel) [inline aldryn_bootstrap3/cms_plugins.py]: Yep there will be no confluct

Comment (stefanfoulis) [inline aldryn_bootstrap3/cms_plugins.py]: yeah, that is fine. It is just a label. And this plugin is in the "Bootstrap3" section, so it won't be confused with the plugin in "Filer"/"File"

File: aldryn_bootstrap3/models.py [hunk 2]
```diff
@@ -309,6 +309,47 @@ def srcset(self):
...
+        on_delete=models.SET_NULL,
+        related_name='+',
+    )
+    name = model_fields.MiniText(
+        _("name"),
+        blank=True,
+        default='',
+    )
+    open_new_window = models.BooleanField(default=False)
+    show_file_size = models.BooleanField(default=False)
```
Comment (stefanfoulis) [inline aldryn_bootstrap3/models.py]: All fields should also have a translatable ``verbose_name``. But that is a problem that most existing models in this app have and should be addressed in a separate pull-request. So no need to fix it here and now.

**Comment by stefanfoulis** [comment]
:+1:

File: aldryn_bootstrap3/models.py (hunk 2 above)
Comment (FinalAngel) [inline aldryn_bootstrap3/models.py]: alright I'll create an issue

**Comment by cyberstar** [comment]
:+1:

**Pull request merged**